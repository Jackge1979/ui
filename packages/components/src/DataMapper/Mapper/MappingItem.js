import React from 'react';

function display(item) {
  return '[ ' + item.source + ' => ' + item.target + ' ]'
}

export default function MappingItem({mappingItem}) {
  return (
    <div id='mappingItem'>{display(mappingItem)}</div>
  )
}
