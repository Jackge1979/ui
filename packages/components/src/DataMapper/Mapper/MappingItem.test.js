import React from 'react';
import MappingItem from './MappingItem.js';
import renderer from 'react-test-renderer';

const item = ['element_in', 'element_out'];

it('single-item', () => {
	// create React tree
	const tree = renderer.create(<MappingItem mappingItem={item} />).toJSON();
	expect(tree).toMatchSnapshot();
});
