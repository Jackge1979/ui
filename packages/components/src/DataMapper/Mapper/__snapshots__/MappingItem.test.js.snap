// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`single-item 1`] = `
<div
  id="mappingItem"
>
  [ undefined =&gt; undefined ]
</div>
`;
